cfn-template/deeplearning.template

```diff
@@ -103,44 +167,57 @@
       "Description" : "The Linux mount point for the EFS volume",
       "Type": "String",
       "MinLength": "1",
-      "Default": "myEFSvolume"
+      "Default": "efs"
+    },
+    "EFSServesData" : {
+      "Description" : "Use EFS for serving data to workers",
+      "Type": "String",
+      "Default": "false",
+      "AllowedValues": ["true", "false"]
+    },
+    "EBSOptimized" : {
+      "Description": "Is the instance EBS optimized? Not all instace types support EBS optimized option.",
+      "Type": "String",
+      "Default": "true",
+      "AllowedValues": [ "false", "true"]
+    },
+    "ActivateCondaEnv" : {
```

Comment:
This will benefit people who use conda environment, but it's better to keep this as an optional, given that we allow using customized AMI so people might also use base AMI or other AMI which does not install Conda then this will fail.